### PR TITLE
Handling cancel image upload 373

### PR DESCRIPTION
**Pull request by zachneill**
Cancelling or removing image removes the image from the peewee database and deletes the file from the folder. Fixes the issue where you have to click remove, refresh the page, then click remove again for the file to be removed. Also actually deletes the file, whereas it would previously just be hidden. Cancelling also works. 

Added multipart/form-data to buildingManagement.html
Used same JS function method for either removing or cancelling. 
Added os remove function to delete the file. 
Resets roomImageURL to null.

How to test: 
Upload an image and remove it, You may refresh the page and check the database and file directory. 
Or, upload a large file and cancel it before it uploads completely.

### CODE REVIEW

**Comment by zachneill** [comment]
[CAS #373: Cancelling an image upload in building management is not handled](https://trello.com/c/RiX7rsoJ/233-cas-373-cancelling-an-image-upload-in-building-management-is-not-handled)

File: app/controllers/main_routes/buildingManagement.py [hunk 1]
```diff
@@ -62,14 +62,14 @@ def getRoomData(rID):
...
-        file = request.files['file']
-        filename = secure_filename(file.filename)
-        file.save(os.path.join('app/static/images/', filename))
-    except Exception as e:
-        print(e)
-        return ("Image failed to upload to the server", 500)
+    file = request.files['file']
+    filename = secure_filename(file.filename)
+    file.save(os.path.join('app/static/images/', filename))
+    # except Exception as e:
```
Comment (bryantal) [inline app/controllers/main_routes/buildingManagement.py]: Are these supposed to be commented out?

**Comment by BrianRamsay** [comment]
@zachneill , there's a comment from Alex not addressed. Can you get this one squared away?

**Review by bryantal — CHANGES_REQUESTED** [review]
Hmm I don't think we want that image in development

**Review by bryantal — APPROVED** [review]

**Review by BrianRamsay — APPROVED** [review]

**Pull request merged**